src/api/lib/data.ts

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs'
+import { DB, ServiceKey } from '../types/admin'
+
+const data: DB = {
+  serviceKeys: []
+}
+
+const getData = (serviceKeys: string) => {
+  if (serviceKeys) {
+    fs.readFile('./data.db', (err, file) => {
+      if (err) {
+        return console.log('error', err)
+      }
+      console.log('file read success')
+      const fileData: DB = JSON.parse(file.toString() || '{}')
+      data.serviceKeys = fileData.serviceKeys || []
+    })
+  }
+}
+
+const setData = (serviceKeys: string, data: DB) => {
```

Comment:
`serviceKeys` is a bad name for the feature. This file is meant to be used for all use cases.

```
const setData = (table: string, records: any) => {
  data[table] = records
  fs.writeFile('./data.db', data, () => {})
}
```